CMakeLists.txt

```diff
@@ -255,8 +258,8 @@ if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
 endif(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
 
 # Create the sixtrack binary name string
-set(SIXTRACK_BINARY_NAME    "${PROJECT_NAME}_${NUM_VERSION}_${SIXTRACK_FEAT_CR}")
-set(SIXTRACK_BINARY_NAME_DA "SixDA_${NUM_VERSION}_${SIXTRACK_FEAT_DA}")
+set(SIXTRACK_BINARY_NAME    "${PROJECT_NAME}_${NUM_VERSION}${GIT_BUILD}_${SIXTRACK_FEAT_CR}")
```

Comment:
Why no separator between `${NUM_VERSION}` and `${GIT_BUILD}`?

Reply:
There is a separator. It's a dash, which is the standard for build "numbers".